### PR TITLE
Mining things

### PR DESCRIPTION
**Pull request by NULL882**
## About the PR

For https://github.com/space-wizards/space-station-14/issues/14226.

Added mining structures for mining pods, shuttles, etc. for Lavaland Station.
## Media

usage example:
image
mining window:
image
mining tiles:
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by UbaserB** [comment]
Can i see what it looks like with FoV and lighting?

**Comment by BolloTea** [comment]
Corner windows please.  Many mappers simply need this. Thank you in advance

**Comment by deltanedas** [comment]
shouldnt have a cl since not visible to players

File: Resources/Textures/Tiles/attributions.yml [hunk 1]
```diff
@@ -109,4 +109,9 @@
...
   license: "CC-BY-NC-SA-3.0"
   copyright: "taken at commit 0587dd16e28108bdf0b0a28e2caae4319845e861, and recolored by TheShuEd"
-  source: "https://github.com/Mojave-Sun/mojave-sun-13"
\ No newline at end of file
+  source: "https://github.com/Mojave-Sun/mojave-sun-13"
+
+- files: ["mining_floor.png", "mining_floor_dark.png", "mining_floor_light.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "taken at https://github.com/ParadiseSS13/Paradise/commit/43889a89d5a9378fd120d627f74613edb1841a66"
+  source: "https://github.com/ParadiseSS13/Paradise"
```
Comment (metalgearsloth) [inline Resources/Textures/Tiles/attributions.yml:117]: These don't looke like they're in goonstation folders hence wouldn't be NC assets.

File: Resources/Textures/Structures/Doors/Airlocks/Glass/mining.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,201 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from https://github.com/ParadiseSS13/Paradise/tree/master/icons/obj/doors/airlocks/survival",
+  "size": {
```
Comment (metalgearsloth) [inline Resources/Textures/Structures/Doors/Airlocks/Glass/mining.rsi/meta.json:5]: This should point to a commit and not a branch.

File: Resources/Textures/Tiles/attributions.yml (hunk 1 above)
Comment (NULL882) [reply Resources/Textures/Tiles/attributions.yml:117]: They were taken from Paradise Station.

**Comment by NULL882** [comment]
> Diagonal windows please. Many mappers simply need this. Thank you in advance

Maybe in the next PR I'll do it.

File: Resources/Textures/Tiles/attributions.yml (hunk 1 above)
Comment (NULL882) [reply Resources/Textures/Tiles/attributions.yml]: Fixed

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**